app/views/schools/show.html.erb

```diff
@@ -44,6 +44,8 @@
 <% end %>
 
 <% if @show_standard_prompts %>
+  <%= render 'management/schools/dashboard_message', school: @school %>
```

Comment:
I think this needs to be more prominent. This list "standard prompts" is a set of reminders that are more or less always shown. As this message is presumably more urgent, then I think it needs to be higher up the page.

I'd suggest putting it immediately after the @dashboard_alerts section, and ensure its only shown to school users. Should be safe to just wrap it in the test for @show_standard_prompts.

Reply:
Thanks @ldodds for the tips. Have moved it up as suggested and wrapped in a @show_standard_prompts test. I'll merge when the tests have finished running 👍